index.html

```diff
@@ -1,48 +1,249 @@
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
+          <span>
+            <img src="./images/icon-arrow-right.svg" alt="icon arrow"
+          /></span>
+        </div>
+      </div>
+      <img src="./images/laptop-stand.svg" alt="laptop-stand" />
+
+      <div class="skills-box">
+        <h3 class="project-name">Project name goes here</h3>
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
+          <span>
+            <img src="./images/icon-arrow-right.svg" alt="icon arrow" />
+          </span>
+        </div>
+      </div>
+      <img src="./images/laptop-pink.svg" alt="laptop-pink" />
 
+      <div class="skills-box">
+        <h3 class="project-name">Project name goes here</h3>
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
+          <span>
+            <img src="./images/icon-arrow-right.svg" alt="icon arrow" />
+          </span>
+        </div>
+      </div>
+    </section>
+    <section class="about">
       <div>
-        <h1 class="name">Charlie Giorgania</h1>
-        <p class="intro-text">
-          Hello! I am a software developer!<br> I can help you build a product, feature or website.<br> Take a look of
-          my
-          works.<br>If you like what you see and have a<br> project you need coded, don’t hesitate and contact me.</br>
+        <h4>About</h4>
+      </div>
+      <div>
+        <p class="about-text">
+          Lorem ipsum dolor sit amet, consectetur<br />
+          adipiscing elit, sed do eiusmod tempor<br />
+          incididunt ut labore et dolore magna<br />
+          aliqua. Ut enim ad minim veniam, quis<br />
+          nostrud exercitation ullamco laboris nisi<br />
+          ut aliquip ea eiusmod
         </p>
       </div>
-      <div class="start-collab-box">
-        <button type="button">Start collaboration</button>
+      <div class="my-cv-box">
+        <button type="button">
+          Get my CV
+          <span>
+            <img src="./images/icon-download.svg" alt="download" />
+          </span>
+        </button>
```

Comment:
- [ ] Please the button (e.g., "Get my CV") follow the button styling: Active, Hover, and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9))

![b](https://user-images.githubusercontent.com/63972670/166093005-beb49dd8-d487-4c5b-9576-42005fefb861.JPG)